packages/react/src/components/Embed/Embed.tsx

```diff
@@ -165,7 +165,7 @@ class Embed extends AutoControlledComponent<WithAsProp<EmbedProps>, EmbedState>
                 },
               },
             })}
-            {Box.create(iframe, { defaultProps: { as: 'iframe' } })}
+            {Box.create(iframe, { defaultProps: { as: 'iframe', style: { display: 'block' } } })}
```

Comment:
This is the correct fix but I don't like the implementation.
1. not `style` but `styles`,
2. do not write the styles directly here but rather add a new `iframe` slot to `embedStyles` and here just pass `styles.iframe` (For inspiration see the usage of `styles.control` below)